api_tests/nodes/views/test_node_alternative_citation_list.py

```diff
@@ -748,16 +748,21 @@ def test_add_citation_logged_out_private_reg(self):
         assert_equal(res.json['errors'][0]['detail'], 'Authentication credentials were not provided.')
         assert_equal(len(registration.alternative_citations), 0)
 
+
 class TestGetAlternativeCitations(ApiTestCase):
     def request(self, errors=False, is_admin=False, is_contrib=True, logged_out=False, **kwargs):
         admin = AuthUserFactory()
+        registration = kwargs.get('registration', None)
```

Comment:
I think it is supposed to be sending it to create_project explicitly. That was the original test, passing the kwargs along to the create project, I just needed the extra knowledge of if it was a registration to change the URL.